### PR TITLE
NOISSUE - Update Copyright Notice

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Add CI check for non-go files to check that the files contain a license

### Which issue(s) does this PR fix/relate to?
No issue

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes
N/A

### CODE REVIEW

**Review by SammyOina — APPROVED** [review]

**Review by dborovcanin — COMMENTED** [review]
Can [Goheader linter](https://golangci-lint.run/usage/linters/#goheader) be used for this?

**Comment by rodneyosodo** [comment]
Goheader only does it for go files.

File: .github/workflows/check-license.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            --exclude=\*.crt --exclude=\*.key --exclude=\*.pem \
+            --exclude=\*.zed --exclude=\*.hcl \
+            --exclude=CODEOWNERS --exclude=LICENSE --exclude=MAINTAINERS \
+            --exclude=\*.md --exclude=\*.json --exclude=\*.csv \
+            --exclude=\*.mod --exclude=\*.sum \
+            --exclude=\*.tmpl --exclude=\*.args \
+            --regexp "Copyright (c) Abstract Machines" .)" ]; then
+            echo "License header check failed"
+            exit 1
+          fi
```
Comment (dborovcanin) [inline .github/workflows/check-license.yaml:33]: Can we simplify a bit and log the failed file, something like:
```bash
ff=$(grep -rcL --exclude-dir=.git --exclude-dir=build \
            --exclude='*.{crt,key,pem,zed,hcl,md,json,csv,mod,sum,tmpl,args}' \
            --exclude={CODEOWNERS,LICENSE,MAINTAINERS} \
            --regexp "Copyright (c) Abstract Machines" . | head -n 1)

if [ "$ff" ]; then
    echo "License header check failed on file: $ff"
    exit 1
fi
```

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by ianmuchyri — APPROVED** [review]

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Update headers for Domains in SDK and CLI.

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**